definition-importer/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM openjdk:17-buster
 
 ENV FHIR_BASE=http://localhost:8080/fhir
+ENV FHIR_VERSION=dstu3
 ARG TARGET_DIR=hapi-fhir-cli
 ARG HAPI_FHIR_CLI_ARCHIVE=hapi-fhir-5.3.0-cli.tar.bz2
```

Comment:
Did you try to use `ADD` to automatically download the archive?

Reply:
Yeah I couldn't find a way to make the downloading automatic. The URL I'm referring to in the readme redirects the browser to a different url where the file is actually located. If I try to curl the link https://github.com/hapifhir/hapi-fhir/releases/download/v5.3.0/hapi-fhir-5.3.0-cli.tar.bz2 it will just download a webpage that says I will be redirected to a different page.

Reply:
try `curl -LJO <url>`, it'll download the file itself

Reply:
awesome I didn't know that. I have added that